docker-compose.override.yml

```diff
@@ -12,3 +12,8 @@ services:
       target: dev
     volumes:
       - ./core/graphelier-service:/go/src/graphelier/core/graphelier-service
+  graphelier-news:
+    build:
+      target: dev
+    volumes:
+      - ./news/service:/news/service
```

Comment:
Since there's no hot-reload setup for dev I dont see why this is done

Reply:
i set up the hot reload